systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -145,6 +145,7 @@ public static void waitForZkMntr(String namespaceName, String clusterName, Patte
         }
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Why not update it instead of keeping the deprecated method?

Reply:
If I change `getType()` here and change it to `getName() , Should I remove it from the other places also and also from the tests ?

Reply:
I guess - of course not where the `getType` is specifically what the test tests. But I don't know what the other places are. I think I commented mainly on those where I saw the deprcation suppression.,